code/src/OSFramework/Helper/AsyncInvocation.ts

```diff
@@ -4,4 +4,8 @@ namespace OSFramework.Helper {
     export function AsyncInvocation(callback: Callbacks.Generic, ...args: any) {
         if (callback) setTimeout(() => callback(...args), 0);
     }
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+    export function SyncInvocation(callback: Callbacks.Generic, ...args: any) {
+        if (callback) callback(...args);
+    }
```

Comment:
Isn't this a bit overkill? Or you like the idea of isolating the method, in order to test if the handler exists?

Reply:
I isolated the method, so I wouldn't have to change every call to the AsyncInvocation. But if you prefer, I can do that.